pkg/collector/corechecks/servicediscovery/model/model.go

```diff
@@ -0,0 +1,32 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+// Package model contains the types for the service_discovery check.
+package model
```

Comment:
It's not idiomatic Go to have a separate model package. I think we might do that in other places in the agent, unfortunately.

Reply:
Yes, we do in other places of the agent, but we don't necessarily need to do it here. Would it be better to be moved to the `servicediscovery` package?

Reply:
Moved to `servicediscovery` package.

Reply:
ended up adding it back because of import cycle 😢